packages/fxa-auth-server/lib/senders/templates/subscriptionDowngrade.html

```diff
@@ -0,0 +1,41 @@
+{{> subscriptionIcon}}
+
+ <table class="featured-story featured-story--top" cellspacing="0" cellpadding="0">
```

Comment:
Not scope for this PR, but I wonder if there are other common parts in the subscription based emails that might be able to be pulled into the partials directory.

Reply:
Yeah, I was thinking next sprint, when I should be adding the last of these new emails, I'd take a polish run through and try to dry up the repetition between these templates now that I see how similar they all are

Reply:
https://jira.mozilla.com/browse/FXA-1639